recipe/install_ucx.sh

```diff
@@ -17,6 +17,7 @@ fi
     --prefix="${PREFIX}" \
     --with-sysroot \
     --disable-static \
+    --enable-openmp \
```

Comment:
@jakirkham I am a bit confused about what this does -- UCX already links to libgomp without this before ( see https://github.com/conda-forge/ucx-split-feedstock/pull/173#discussion_r1582456885).

Reply:
Am just making it explicit (explicit is better than implicit)

Also this is a reminder that we should change this to `--disable-openmp` in the future

Reply:
> explicit is better than implicit

like it!